stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml

```diff
@@ -84,6 +84,11 @@ spec:
         - '/bin/sh'
         - '-c'
         - >
+          {{- if .Values.artifactory.userPluginSecrets }}
+          echo "Copying plugins";
+          mkdir -p {{ .Values.artifactory.persistence.mountPath }}/etc/plugins;
+          cp -Lrf /tmp/plugin/*/* {{ .Values.artifactory.persistence.mountPath }}/etc/plugins/;
```

Comment:
@jainishshah17 - I think this actually breaks support for providing the cluster license via a Kubernetes secret. The `entrypoint-artifactory.sh` file contains the following logic that only copies files from the `/artifactory_extra_conf` directory into `$ARTIFACTORY_DATA/etc/` if the latter doesn't exist:

```bash
    if [ ! -d ${ARTIFACTORY_DATA}/etc ]; then
        mkdir -pv ${ARTIFACTORY_DATA}/etc || errorExit "Failed creating ${ARTIFACTORY_DATA}/etc"

        # Add extra conf files to a newly created etc/ only!
        addExtraConfFiles
    fi
```

Since this effectively puts the plugins in that directory, the license isn't copied

Reply:
Here are the relevant logs from our primary node:

```
2019-04-03 15:19:07  [262 entrypoint-artifactory.sh] Setting up Access data directories if missing
2019-04-03 15:19:07  [152 entrypoint-artifactory.sh] Adding extra configuration files to /var/opt/jfrog/artifactory/access/etc if any exist
```

As you can see, it skips over the code block below that's intended to copy the files from `/artifactory_extra_conf`:

```
# Add additional conf files that were mounted to ARTIFACTORY_EXTRA_CONF
addExtraConfFiles () {
    logger "Adding extra configuration files to ${ARTIFACTORY_HOME}/etc if any exist"

    # If directory not empty
    if [ -d "${ARTIFACTORY_EXTRA_CONF}" ] && [ "$(ls -A ${ARTIFACTORY_EXTRA_CONF})" ]; then
        logger "Adding files from ${ARTIFACTORY_EXTRA_CONF} to ${ARTIFACTORY_HOME}/etc"
        cp -rfv ${ARTIFACTORY_EXTRA_CONF}/* ${ARTIFACTORY_HOME}/etc || errorExit "Copy files from ${ARTIFACTORY_EXTRA_CONF} to ${ARTIFACTORY_HOME}/etc failed"
    fi
}
```

Reply:
CC @eldada @rimusz

Reply:
@eldada @rimusz @robbie-demuth Check out https://github.com/jfrog/charts/pull/289

We noticed this breaks a bunch of stuff for a fresh install because of the exact check you pointed out.

@properpanda submitted https://github.com/jfrog/charts/pull/289 and issue https://github.com/jfrog/charts/issues/287 for it.